.github/ISSUE_TEMPLATE/docs-issue-template.md

```diff
@@ -18,7 +18,6 @@ assignees: ''
 - [ ] Item dependencies have been identified
 
 ## DoD
-- [ ] Documentation has been created/updated (if applicable)
-- [ ] Migration Guide has been created/updated (if applicable)
 - [ ] Relevant stories (Features, A11y) are created/updated
```

Comment:
I suggest adding another checkbox along the lines of `Implementation delivers all acceptance criteria defined in the ticket`

Reply:
Since the goal of this template is to make sure that you did not forget anything, I think it's valid to include this point because it's another reminder for you to double check that all ACs are delivered. Maybe in a simple ticket this is not worth it, but in a more complex one where there are multiple ACs I believe it makes sense to once again validate that everything described in the ticket is being tackled. 
As I said, for me it makes sense to include, but would like to get other perspectives from the team.